desktop_version/src/editor.cpp

```diff
@@ -2155,7 +2155,12 @@ bool editorclass::save(std::string& _path)
         scriptString += script_.name + ":|";
         for (size_t i = 0; i < script_.contents.size(); i++)
         {
-            scriptString += script_.contents[i] + "|";
+            scriptString += script_.contents[i]
+
+            // Inserts a space if the line ends with a :
+            + (script_.contents[i].length() != 0 && *script_.contents[i].rbegin() == ':' ? " " : "")
+
+            + "|";
```

Comment:
This works, but I'd rather have this factored out into a full-fledged if-condition instead of being shoved into a ternary. Also, it should just be `script_.contents[i].length()` instead of having that unnecessary `!= 0` at the end. (That way you don't have to think about the difference between `!= 0` and `> 0`.)